.changeset/late-wasps-nail.md

```diff
@@ -0,0 +1,17 @@
+---
+'@xstate/vue': minor
+---
+
+Added new `useActor`, which is a composable that subscribes to emitted changes from an existing `actor`:
+
+```js
+import { useActor } from '@xstate/vue';
+
+export default defineComponent({
+  props: ['someSpawnedActor'],
+  setup() {
+    const { state, send } = useActor(someSpawnedActor);
```

Comment:
Not 100% sure but I would expect this to come from props:

```suggestion
  setup({ someSpawnedActor }) {
    const { state, send } = useActor(someSpawnedActor);
```

Reply:
Oops. Sorry, you are totally correct. But better not to destructure props because you'll loose Vue's reactiviy. Should be:
``` js
setup(props) {
    const { state, send } = useActor(props.someSpawnedActor);
    ...
}
```